ecommerce-ui/src/App.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+
+import './App.css';
+import logo from './logo.svg';
+
+import Location from './components/Location';
+import airbandbs from './data/airbandbs.json';
+
+// console.log({airbandbs});
+
```

Comment:
need to have state here:
   constructor(props) {
        super(props);
        this.state = {
            rentals: []
        };
        this.handleAdd = this.handleAdd.bind(this);
        this.handleRemove = this.handleRemove.bind(this);
    }